pages/index.tsx

```diff
@@ -56,6 +64,35 @@ export default function Home(props: {
     }
   });
 
+  var storage = firebase.storage();
+  // Change bucketName to match firebase storage bucket for speakers
+  var bucketName = 'hackportal-speaker-images';
```

Comment:
Not sure if it's a great idea to leave `bucketName` hardcoded into the codebase like that

Reply:
It isn't.